docs: use new `nuxi module add` command in installation (#6)
hamlogic/nuxt-cloudflare-analytics: README.md

## Setup

> - Add `nuxt-cloudflare-analytics` dependency using yarn or npm to your project `npx nuxi@latest module add cloudflare-analytics

Author: ```suggestion
- Add `nuxt-cloudflare-analytics` dependency to your project with `npx nuxi@latest module add cloudflare-analytics`
```
